### ISSUE TITLE
StartAdbInteractor with default values doesn't work on Windows

### ISSUE THREAD

**Issue by joffrey-bion** [Malinskiy/adam#41]
**Describe the bug**
Running `StartAdbInteractor().execute()` never successfully starts an adb server on Windows, because the code explicitly searches for a file named `adb` in the Android home, but on Windows it's called `adb.exe`.

**To Reproduce**
Simply run `StartAdbInteractor().execute()` (with default values) on Windows, the result is always `false`.

**Expected behavior**

It would be great that the default `adb` binary path was OS-dependent and was `adb.exe` on Windows instead of `adb`, so it would work as smoothly as on other OS'es.
An even simpler option would be to fallback to a `ProcessBuilder` with just `adb` as command name (not a full path), because on my machine `adb` is on the `PATH`, so the filename doesn't matter.

**Workaround**

The following works fine:
```kotlin
    val androidHome = System.getenv("ANDROID_HOME")?.ifBlank { null }
    val adbBinary = androidHome?.let { Paths.get(androidHome, "platform-tools", "adb.exe")}
    val adbServerStarted = StartAdbInteractor().execute(adbBinary = adbBinary?.toFile())
    if (!adbServerStarted) {
        error("No adb server running")
    }
```

**Android device (please complete the following information):**
 - Device: [e.g. Nexus 5] *irrelevant*
 - OS: [e.g. Android 8.1 official] *irrelevant*
 - Adam Version: 0.2.5
- Android Debug Server Version [e.g. 41]: Android Debug Bridge version 1.0.41, Version 31.0.2-7242960

**Comment by Malinskiy** [comment]
Hey @joffrey-bion

Thanks for submitting the issue. I don't have access to a Windows machine, so your feedback is important.

The only reasonable way to test this is to add another CI configuration that will test this behaviour on an actual Windows instance. Although windows instances will not support running emulator, it should be possible to run at least the adb binary. Would you be able to contribute to this?

Thanks,
Anton

**Comment by Malinskiy** [comment]
Alternatively please test #42 and reply here if it fixes the problem for you

**Comment by joffrey-bion** [comment]
Sorry for the delay, this change is really similar so I believe it should work.
I will try to run this on my machine, and report here

**Comment by Malinskiy** [comment]
Since I don't have an environment to verify this change I'll await your reply. Thank you!

**Comment by joffrey-bion** [comment]
Sorry for the delay, this changes works fine as expected :)

**Comment by Malinskiy** [comment]
Closed by #42

**Comment by Malinskiy** [comment]
@joffrey-bion released in 0.3.1
